README.md

```diff
@@ -1,7 +1,141 @@
+>
+> The reason behind these changes is to focus Mattermost developer resources on improving the platform’s performance and core features to ensure Mattermost continues being resilient, stable, and best-in-breed for critical operations.
+>
+> ️💡 [Learn more](https://forum.mattermost.com/t/upcoming-product-changes-to-boards-and-various-plugins/16669)
+
+# Mattermost boards plugin 
+
+### A self-hosted plugin for managing boards withing a Mattermost installation
+
+Focalboard is an open source, multilingual, self-hosted project management tool that's an alternative to Trello, Notion, and Asana.
 
 **[Mattermost Boards](https://mattermost.com/boards/)** is the Mattermost plugin version of Focalboard that combines project management tools with messaging and collaboration for teams of all sizes. To access and use **Mattermost Boards**, install or upgrade to Mattermost v6.0 or later as a [self-hosted server](https://docs.mattermost.com/guides/deployment.html?utm_source=focalboard&utm_campaign=focalboard) or [Cloud server](https://mattermost.com/get-started/?utm_source=focalboard&utm_campaign=focalboard). After logging into Mattermost, select the menu in the top left corner of Mattermost and select **Boards**.
 
-***Mattermost Boards** is installed and enabled by default in Mattermost v6.0 and later.*
+![Focalboard](website/site/static/img/hero.jpg)
+
+## Try Focalboard
+
+### Mattermost Plugin
+
+After downloading and installing the plugin in the System Console, select the menu in the top left corner and select **Boards**. Access the latest releases of the focalboard plugin by downloading the `mattermost-plugin-focalboard.tar.gz` file from the releases in this repository: <https://github.com/mattermost/focalboard/releases>
+
+
+### API Docs
+
+Boards API docs can be found over at <https://htmlpreview.github.io/?https://github.com/mattermost/focalboard/blob/main/server/swagger/docs/html/index.html>
+
+### Getting started
+
+Our [developer guide](https://developers.mattermost.com/contribute/more-info/focalboard/mattermost-boards-setup-guide/) has detailed instructions on how to set up your development environment for the **Mattermost boards plugin**. You can also join the [~Focalboard community channel](https://community.mattermost.com/core/channels/focalboard) to connect with other developers.
+
+Clone [mattermost-server](https://github.com/mattermost/mattermost-server) into sibling directory.
+
+You also want to have the environment variable `MM_DEBUG"true"` set, otherwise the plugin
+will be compiled for Linux, Windows, and Darwin ARM64 and x64 architecture every single time. Setting
+the `MM_DEBUG` to `true` makes the plugin compile and build only for the OS and architecture
+you are building on.
+
+In your Mattermost configuration file, ensure that `PluginSettings.EnableUploads` is set to `true`, and `FileSettings.MaxFileSize` is
```

Comment:
Should admins receive this guidance **before** attempting to install the plugin binary? Or are released binaries small enough to easily fit within the maximum file size default value?

Reply:
This is for local setup only and can also be changed from the system console.